src/microbe_stage/Patch.cs

```diff
@@ -116,6 +117,7 @@ public int GetSpeciesPopulation(Species species)
 
     public override string ToString()
     {
-        return $"Patch \"{Name}\"";
+        return string.Format(CultureInfo.CurrentCulture,
```

Comment:
I think this is 100% used just when debugging C# code, this should not call translation services.